pkgs/applications/blockchains/btcpayserver/default.nix

```diff
@@ -1,7 +1,9 @@
 { lib, stdenv, fetchFromGitHub, fetchurl, linkFarmFromDrvs, makeWrapper,
-  dotnetPackages, dotnetCorePackages
+  dotnetPackages, dotnetCorePackages, altcoinSupport ? false
 }:
 
+with lib;
+
```

Comment:
```suggestion
```

Reply:
Fixed in bf6cac3e7a94de1a5dc0da81899ec489780210a7